UNITTESTS/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ set(TEST_SUITES)
 
 # Get all matched tests.
 file(GLOB_RECURSE unittest-file-list
-  "unittest.cmake"
+  "../unittest.cmake"
```

Comment:
Note: The glob syntax is a bit special - this pattern matches _any levels_ of subdirectories under `..` with my version of CMake. Let's see if it works with our CI...